modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/src/main/resources/site-initializer/resource-permissions.json

```diff
@@ -809,5 +848,15 @@
 		"resourceName": "com.liferay.object.model.ObjectDefinition#[$OBJECT_DEFINITION_ID:Suite$]",
 		"roleName": "Testray User",
 		"scope": 1
+	},
+	{
+		"actionIds": [
+			"UPDATE",
+			"VIEW"
+		],
+		"primKey": "0",
+		"resourceName": "com.liferay.portal.kernel.model.User",
+		"roleName": "Testray User",
+		"scope": 1
```

Comment:
This "UPDATE" permission worries me a little, did you checked if applying this permission it's possible to update user from someone else?

The `Testray Administrator` is the only one that can UPDATE users, we must check this before move-on